docs/documentation_website.md

```diff
@@ -0,0 +1,21 @@
+---
+nav_order: 10
+---
+
+# This documentation website
+
```

Comment:
Maybe some preamble about where documentation is saved (perhaps linking to other sources, Confluence, Sharepoint, etc.)

Reply:
Same as above - think that's out of scope here but we could add a new page for it if you think it would be valuable?

Reply:
In which case, maybe just have a single title for this page.